truffle/contracts/Auction.sol

```diff
@@ -119,7 +119,7 @@ contract Auction is ReentrancyGuard {
             );
         }
         require(
-            msg.value + bids[msg.sender] > highestBid,
```

Comment:
- User A bids 1, 2, 3 ETH --> bids[A] = 1 + 2 + 3 = 6
- User B bids 4 ETH --> bids[B] = 4 < 6 --> L122 will fail although B is the current highest bidder